paasta_tools/metrics/metastatus_lib.py

```diff
@@ -267,7 +427,23 @@ def assert_tasks_running(metrics):
     )
 
 
-def assert_no_duplicate_frameworks(state: Dict, framework_list: List[str]=['marathon', 'chronos']):
+def assert_kube_pods_running(
+    kube_client: KubeClient,
+) -> HealthCheckResult:
+    statuses = [get_pod_status(pod) for pod in get_all_pods(kube_client)]
+    running = statuses.count(PodStatus.RUNNING)
+    pending = statuses.count(PodStatus.PENDING)
+    failed = statuses.count(PodStatus.FAILED)
+    return HealthCheckResult(
+        message=f'Pods: running: {running} pending: {pending} failed: {failed}',
+        healthy=True,
```

Comment:
Should we maybe only return true if at least 1 pod is running? Do we do something similar for marathon apps iirc?